recipes/radtk/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: radtk
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage("radtk", max_pin="x.x") }}
+
+source:
+  url: https://github.com/COMBINE-lab/radtk/archive/v{{ version }}.tar.gz
+  sha256: 116c9b05576de3e9708d2d7817d7b4f4858f9edf44d2a9bb0548159f0ccff6fb
+
+requirements:
+  build:
+    - autoconf
+    - make
+    - {{ compiler('cxx') }}
+    - {{ compiler('rust') }}
+    - pkg-config
+  host:
+  run:
+
+test:
+  commands:
+    - radtk --help || test $? -eq 1
+
+about:
+  home: https://github.com/COMBINE-lab/radtk
+  license: BSD 3-Clause
```

```suggestion
  license: BSD-3-Clause
```
https://spdx.org/licenses/BSD-3-Clause.html